index.d.ts

```diff
@@ -25,11 +25,11 @@ export interface MainProcessIpc extends IpcMain {
 	})();
 	```
 	*/
-	callRenderer<T>(
+	callRenderer<T, S>(
```

Comment:
I think we should use more descriptive type names than `T` and `S` now that there are multiple.